Playground/js/monacoCreator.js

```diff
@@ -86,12 +86,25 @@ class MonacoCreator {
     };
 
     setupDefinitionWorker(libContent) {
-        this.definitionWorker = new Worker('js/definitionWorker.js');
-        this.definitionWorker.addEventListener('message', ({ data }) => {
-            this.deprecatedCandidates = data.result;
-            this.analyzeCode();
-        });
-        this.definitionWorker.postMessage({ code: libContent });
+        // What is the reason behind this worker?

```

Comment:
This is something we need to restore as it is used to analyze the code looking for @deprecated flags

Reply:
Ok so we chatted offline and actually this worker was dl a 7MB file which gives us a very bad score on Google Search